jdbcSource/src/main/java/io/vantiq/extsrc/jdbcSource/JDBCCore.java

```diff
@@ -234,6 +240,9 @@ public void executePublish(ExtensionServiceMessage message) {
         } catch (VantiqSQLException e) {
             log.error("Could not execute requested query.", e);
             log.debug("Request was: {}", request);
+        } catch (Exception e) {
+            log.error("An unexpected error occured when executing the requested query.", e);
+            log.debug("Request was: {}", request);
```

Comment:
Do you not want the request and/or the result when an error occurs? That is, why is this debug vs. error?